src/systems/userinput/bindings/oculus-go-user.js

```diff
@@ -243,15 +261,25 @@ export default function generate3DOFTriggerBindings(device) {
       },
       {
         src: {
-          value: touchpadX,
+          value: touchpadXForPen,
           touching: touchpadButton.touched
         },
         dest: { value: paths.actions.cursor.scalePenTip },
-        xform: xforms.touch_axis_scroll(-0.1)
+        xform: xforms.touch_axis_scroll(0.1, 0.1),
+        priority: 200
       },
       {
         src: {
-          value: dpadCenterStrip,
+          value: touchpadYForPen,
+          touching: touchpadButton.touched
+        },
+        dest: { value: paths.actions.cursor.modDelta },
+        xform: xforms.touch_axis_scroll(1, 0.1),
+        priority: 200
+      },
+      {
+        src: {
+          value: dpadBottomCenterStrip,
```

Comment:
Would be cool to be able to teleport while holding an object by using the top of the touchpad, since here we are dropping with the bottom + center.

Reply:
we're only using the bottom center strip when holding the pen (since we use north to undo) -- non-pen mode uses the full strip.